README.md

```diff
@@ -1 +1,46 @@
-# Trial-Review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Trial-Review
+
+> HEllo-Microverse is a project where we get to review a pull request of our own just to practice how to do reviews on a pull request.
```

Comment:
[OPTIONAL] This is not the Hello-Microverse project this is the Trial-Review one let's change it in this line